<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QMainWindow" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация / Регистрация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Добро пожаловать!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18pt; font-weight:bold;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputLogin">
      <property name="placeholderText">
       <string>Логин</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputPassword">
      <property name="placeholderText">
       <string>Пароль</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputPhone">
      <property name="placeholderText">
       <string>Телефон (при регистрации)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputEmail">
      <property name="placeholderText">
       <string>Email (при регистрации)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelError">
      <property name="text">
       <string></string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnLogin">
        <property name="text">
         <string>Авторизоваться</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRegister">
        <property name="text">
         <string>Зарегистрироваться</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelHint">
      <property name="text">
       <string>admin:admin123</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight</set>
      </property>
      <property name="styleSheet">
       <string notr="true">color:gray; font-size:8pt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>